wagtailio/services/templates/services/blocks/divider.html

```diff
@@ -0,0 +1 @@
+<hr class="divider">
```

Comment:
Some missing newlines at the end of these files

Reply:
Added in aae62057a07a5783934acf586d96f3178315ecb3